doc/howto/images_manage.md

```diff
@@ -67,6 +67,13 @@ To edit the full image properties, including the top-level properties, enter the
 
     lxc image edit <image_ID>
 
+## Delete an image
+To delete an image on a server, enter the following command:
+
+    lxc image delete <image_ID>
+
+Deleting an image won't affect running instances that are already using it, but it will prevent you from launching new instances based on that image.
```

Comment:
I think we should also clarify here that "but it will prevent you from launching new instances based on that image." is only true for manually created images (i.e non-cached ones) and if instead a cached one is deleted it will simply be re-downloaded again on next use.